hazelcast/src/test/java/com/hazelcast/map/BasicMapTest.java

```diff
@@ -565,41 +568,45 @@ public void testMapLockAndUnlockAndTryLock() throws Exception {
         final CountDownLatch latch1 = new CountDownLatch(1);
         final CountDownLatch latch2 = new CountDownLatch(1);
         final CountDownLatch latch3 = new CountDownLatch(1);
+        final CountDownLatch latch4 = new CountDownLatch(1);
+
         Thread thread = new Thread(new Runnable() {
             public void run() {
                 try {
                     check1.set(map.tryLock("key0"));
-                    check2.set(map.tryLock("key0", 3000, TimeUnit.MILLISECONDS));
                     latch0.countDown();
 
-                    map.put("key1", "value1");
+                    check2.set(map.tryLock("key0", timeout, SECONDS));
```

Comment:
the diff view is quite confusing here. the change is actually quite simple. 
I added a new latch -  `latch0`. this shifted all existing latches. so the old `latch0` become `latch1`, the old `latch1` become `latch2`, etc..